dandiapi/api/admin.py

```diff
@@ -97,6 +97,23 @@ class DandisetAdmin(GuardedModelAdmin):
     readonly_fields = ['identifier', 'created']
     inlines = [VersionInline]
 
+    @admin.action(description='Ingest selected dandiset zarr archives')
+    def ingest_dandiset_zarrs(self, request, queryset):
+        for dandiset in queryset:
+            ingest_dandiset_zarrs(dandiset_id=dandiset.id)
+
+        # Return message
+        plural = 's' if queryset.count() > 1 else ''
+        self.message_user(
+            request,
+            f'Ingesting zarr archives for {queryset.count()} dandiset{plural}.',
+            messages.SUCCESS,
+        )
```

Comment:
why not use ngettext here?

Reply:
It just seemed more messy than it was necessary in this case, Since the only change is `dandiset(s)`. However, if you think it's worthwhile to use `ngettext` here as well, I can change it.